typescript/libs/view_helpers.py

```diff
@@ -106,8 +106,9 @@ def is_typescript(view):
     except:
         return False
 
-    return (view.match_selector(location, 'source.ts') or
-            view.match_selector(location, 'source.tsx'))
+    is_ts_file = view.match_selector(location, 'source.ts') or view.match_selector(location, 'source.tsx')
+    is_js_file = view.match_selector(location, 'source.jsx') and cli.enable_language_service_for_js
```

Comment:
I think adding the selector for `source.js` beside `source.jsx` would be better. Like mentioned in my other comment.